src/library/Uno.Material/MaterialResourcesV2.cs

```diff
@@ -56,5 +143,24 @@ private void ExportImplicitStyles(bool value)
 			// but, we can add them through merged-dict instead.
 			this.MergedDictionaries.Add(implicitResources);
 		}
+
+		private Style FindStyle(string key)
+		{
+			if (!this.TryGetValue(key, out var resource) || !(resource is Style style))
+			{
+				// uwp: If the {key} style is clearly defined in {info.Source}, but we can't find it here.
```

Comment:
```suggestion
				// uwp: If the {key} style is clearly defined, but we can't find it here.
```
the context has since changed